css/main.css

```diff
@@ -58,7 +71,10 @@ h2 {
   font-size: 36px;
   line-height: 56px;
 }
-.section-1, .section-3, .section-4 {
+.section-1, 
+.section-3, 
+.section-4, 
+.section-5 {
   display: flex;
   align-items: center;
```

Comment:
Maybe it will be better to create common class for all the sections? For example `.section`.